backend/app/.env

```diff
@@ -1,5 +1,5 @@
 COMPOSE_CONVERT_WINDOWS_PATHS=1
 PORT=4080
-MONGOURI=mongodb://mongo/safetrace
+MONGOURI=mongodb://localhost:10975/safetrace

```

Comment:
@cmalfesi This change breaks my local install. If you change a parameter like this that requires a different setup, I would ask to document what else needs to be changed, otherwise it becomes problematic as we are troubleshooting one thing, we break another one. Thanks! 🙏